static/include/components/apis/generated/movement_sensor-table.md

```diff
@@ -0,0 +1,16 @@
+<!-- prettier-ignore -->
+| Method Name | Description |
+| ----------- | ----------- |
+| [`GetLinearVelocity`](/components/movement-sensor/#getlinearvelocity) | Report the current linear velocity in the x, y and z directions (as a 3D vector) in meters per second. |
+| [`GetAngularVelocity`](/components/movement-sensor/#getangularvelocity) | Report the current angular velocity about the x, y and z axes (as a 3D vector) in degrees per second. |
+| [`GetCompassHeading`](/components/movement-sensor/#getcompassheading) | Report the current [compass heading](https://en.wikipedia.org/wiki/Heading_(navigation)) in degrees. |
```

Comment:
No idea how a link snuck in here, maybe double `))`?? Anyways, this one to an external resource is fine to leave here for, maybe even helpful. Will investigate later.